### ISSUE TITLE
HSBC

### ISSUE THREAD

**Issue by diegobot1977** [simple-icons/simple-icons#5310]
**Name:** HSBC
**Website:** https://www.hsbc.com/
**Alexa rank:** 5,512
**Color:** `#db0012`
**Official resources for icon and color:** see below on the HSBC homepage

**Comment by adamrusted** [comment]
Clear SVG version available on the header of the site. Would you like to give this one a go @diegobot1977? Take a look at our [contribution guidelines](https://www.github.com/simple-icons/simple-icons/tree/develop/CONTRIBUTING.md) and see what you think.

**Comment by diegobot1977** [comment]
Yes for sure

**Comment by ericcornelissen** [comment]
Closing this in favor of  #5304. @diegobot1977, please open issues for icon requests only once in the future.
